public_html/wp-content/plugins/wordcamp-participation-notifier/wordcamp-participation-notifier.php

```diff
@@ -112,37 +112,31 @@ protected function is_post_notifiable( $post ) {
 	 * Updates the activity and associations of a profile when the WordPress.org username on a published speaker
 	 * or organizer post changes.
 	 *
+	 * IMPORTANT NOTE: If a draft post is published via block editor, we will have to add badges and activity here instead of `post_published_or_unpublished` method.
```

Comment:
```suggestion
	 * IMPORTANT NOTE: When a draft post is published via the block editor, badges and activity must be managed here instead of in the `post_published_or_unpublished` method.
```

Reply:
Fixed in https://github.com/WordPress/wordcamp.org/pull/125/commits/d4d2d112f6e4d8083cefa51c327d12579e8970d7